unreleased/hostmetrics-rm-direction-process.yaml

```
@@ -0,0 +1,12 @@
+change_type: deprecation
+
+component: hostmetricsreceiver
+
+note: "Remove direction for paging metrics. The feature gate: `receiver.hostmetricsreceiver.removeDirectionAttributeProcessMetrics` can be set to apply the following"
+
+issues: [11819]
+
+subtext:
```

```suggestion
subtext: |
```

Sorry, this is still needed to indicate a multiline string

---

Thanks, I was just getting back to this :D